lib/cell/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cell
-  VERSION = "4.5.0"
```

Comment:
Previous versioning here seems mess 😄  I'm wondering if anyone using `4.5.0` would be able to get updates from `>= 4.2.0 & < 4.5.0`.

Should we release incremental minor version instead ? cc @apotonick

Reply:
We have not released 4.5.0, yet, have we not?

Reply:
Yeah, not from rubygems but in case someone is using `github` repo in Gemfile directly ? I hope there shouldn't be any problems.

Reply:
Good point, but let's go with 4.2.0 and face the consequences :boom: :joy: